src/main/ngapp/accio/src/app/top-bar/top-bar.component.ts

```diff
@@ -6,14 +6,24 @@ import { Component, OnInit } from '@angular/core';
   styleUrls: ['./top-bar.component.css']
 })
 export class TopBarComponent implements OnInit {
-
   constructor() { }
 
-  ngOnInit(): void {
-  }
+  userSignedIn = false;
+  ngOnInit(): void {}
 
-  loggedIn = false;
-  toggleButton() {
-    this.loggedIn = !this.loggedIn;
+  // this needs to be fixed. 
```

Comment:
Update comment to explain what needs to be fixed.

Reply:
Also is there a reason you don't want to fix it before submitting PR?

Reply:
It's a feature that changes the button to say signed out when the user is signed in and visa-versa. I figured the feature is really only important once the Users API is integrated and I didn't want to end up spending too much time trying to fix it before I got the Gallery/Editor pages set up for the main algorithm. It's something I should've just waited to implement at all.